dashboard/test/mailers/teacher_mailer_test.rb

```diff
@@ -58,4 +58,14 @@ class TeacherMailerTest < ActionMailer::TestCase
     assert_match '1001 student accounts were deleted.', mail.body.encoded
     refute_match "#{removed_students.first.name} (#{removed_students.first.username})", mail.body.encoded
   end
+
+  test 'verified teacher email' do
+    teacher = build :teacher, email: 'test@example.com'
+    mail = TeacherMailer.verified_teacher_email(teacher)
+
+    assert_equal I18n.t('teacher_mailer.verified_teacher_subject', locale: 'en-US'), mail.subject
+    assert_equal [teacher.email], mail.to
+    assert_equal ['teacher@code.org'], mail.from
+    assert_match /Hi,/, mail.body.encoded
```

Comment:
anything else I should test here? the email body is static, so there aren't teacher names or anything to check like the other tests in this file

Reply:
This seems reasonable!